flax/experimental/nnx/tests/test_graph_utils.py

```diff
@@ -24,7 +25,7 @@ def test_flatten(self):
     a = {'a': 1, 'b': nnx.Param(2)}
     g = [a, 3, a, nnx.Param(4)]
 
-    state, static = nnx.graph_utils.graph_flatten(g)
+    state, static = nnx.graph_utils.graph_flatten(g)[:2]
```

Comment:
Nit: I think it might be more clear to do 

```
state, static, _ = ...
```

instead, since you already do unpacking.